src/project-tests/survey-form-tests.js

```diff
@@ -90,27 +114,44 @@ export default function createSurveyFormTests() {
                 FCC_Global.assert.isAbove(document.getElementById('number').placeholder.length, 0, 'The number input field\'s placeholder attribute should have a value of some text ');
             });
 
-            it('11. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
+            it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
                 FCC_Global.assert.isNotNull(dropdown, 'There should be a select field with id="dropdown" ');
                 FCC_Global.assert.isAtLeast(getNumOptions(dropdown.options), 2, "Select should contain at least 2 selectable options ");
+                FCC_Global.assert.isAbove(document.querySelectorAll("#survey-form #dropdown").length, 0, 'The select field with id="dropdown" is not inside the form element ');
             })
 
-            it('12. I can select a field from a series of radio buttons.', function() {
-                var radioButtons = $(":radio");
-                FCC_Global.assert.isAtLeast(radioButtons.length, 2, "There should be at least 2 radio buttons ");
+            it('13. I can select a field from a group of radio buttons. The group should have the name "radio-buttons".', function() {
+                var radioButtons = document.querySelectorAll('input[type="radio"][name="radio-buttons"]');
+                FCC_Global.assert.isAtLeast(radioButtons.length, 2, 'There should be at least 2 radio buttons in the "radio-buttons"-group ');
+                FCC_Global.assert.isAtLeast(document.querySelectorAll('#survey-form input[type="radio"][name="radio-buttons"]').length, 2, 'The radio-buttons are not inside the form element ');
+                FCC_Global.assert.strictEqual(radioButtons.length, getNumValues(radioButtons), "All your radio-buttons must have a value-attribute ");
             });
```

Comment:
This is quite a point you have there!
What about just checking all buttons then?
This would then be the last test in Nr. 13:
```
// Check for a value-attribute on all radio Buttons  
var allRadioButtons = document.querySelectorAll('input[type="radio"]');  
FCC_Global.assert.strictEqual(allRadioButtons.length, getNumValues(allRadioButtons), "All your radio-buttons must have a value-attribute ");
```